### ISSUE TITLE
design: UI mockups

### ISSUE THREAD

**Issue by ashcherbakov** [sicpa-dlab/didcomm-gemini#15]
## What
As a team, we should create UI mockups for didcomm.org for every user story, so that we can do implementation based on the mockups

## Why
We need to understand how UI will look like.

## Success Criteria
- Define UI and create UI mockups for every user story
- Consider involvement of a UI designer
- At least the following pages should be designed:
  - Main page (#27)
  - Search/Browse results page (#35, #36) 
  - Protocol page (text + metadata), see https://hackmd.io/d6BQ4yxXQZa9y9ppEruP4Q

**Comment by ashcherbakov** [comment]
@ajslack @dhh1128 
We finished the first version of the UI prototype for all the main pages (web version only so far; mobile - coming soon): [link](https://www.figma.com/proto/YlEZDRezhuEoUUiAzFn22R/didcomm.org?page-id=28%3A0&node-id=1405%3A2112&viewport=2827%2C-466%2C0.5&scaling=min-zoom&starting-point-node-id=1405%3A2112).
Any feedback is appreciated.

**Comment by ajslack** [comment]
Thanks for sharing the prototype, looking good! A few comments:

- On the homepage the inline emojis sit poorly against the line height of the text and I would argue have minimal useful impact there at that scale and reduce readability. They may be more readable as larger icons that stack above the header texts, eg: 

🪄 
**What is DIDComm?**
DIDComm lets people and...

- The colour contrast between the yellow and gradient background is low (3.5) which is not great for accessibility and poor readability for all. I would suggest swapping out the gradient for something more harmonious with the rest of the site. It feels like a little bit of an arbitrary choice at the minute. - A dark mode would be a nice-to-have if there is time, so thinking about how it might switch would be good.

- I notice the text is written in American English, which is fine but we should make sure that it's consistent. Is there an intention to provide multi-language support? Perhaps we could invite community community to submit translations via GitHub once it's up? How then would I switch language, do we need to put a placeholder for that somewhere?

- I find the positioning of the search bar in the centre of the header odd as it jumps from being a page element on the Homepage. This may also be difficult to translate into a consistent mobile experience. I would left align it over the first column of pages.

- The protocol page needs a back button to return to the search results. Persisting search results would be desirable.

- On the protocol page, 'alternates you might prefer' would be great to display as cards like on the homepage 'recently added protocols'

- Curious about the choice of the print button on the protocol info page. Is this a required feature? If we are going to have it then can we also put a reminder for people to think about the environmental impact of printing 🙃.
- More useful action buttons could be in place of the print button, for example a link to (in the case of the json example) the Node website or a 'copy code to clipboard' button so somebody could paste straight into their terminal.

**Comment by ashcherbakov** [comment]
Thank you for feedback.

I will let our UI experts to answer UI-related questions, but can provide some comments about requirements-specific things:

> I notice the text is written in American English, which is fine but we should make sure that it's consistent. Is there an intention to provide multi-language support? Perhaps we could invite community community to submit translations via GitHub once it's up? How then would I switch language, do we need to put a placeholder for that somewhere?

Currently we don't have a requirement for multi-language support, and I believe this is not part for the current milestone (@dhh1128?). We can have a mock-up for the language change, but I think we will not implement it right now.
As for the current text itself (in English), it's mostly take from existing didcomm.org website. I think we may change/adjust it, and must check for the correctness. Probably @dhh1128 may help or guide here.

> On the protocol page, 'alternates you might prefer' would be great to display as cards like on the homepage 'recently added protocols'

The content of that page is supposed to be just a text rendered from the protocol definition submitted as a markdown (protocol definition example: https://hackmd.io/d6BQ4yxXQZa9y9ppEruP4Q). So, we are not supposed to analyse it content, and will show it as-is. The current text on the mock-up is an example which is taken from an npm package just for reference (this is probably confusing).

> Curious about the choice of the print button on the protocol info page. Is this a required feature?

Being able to print the protocol definition is part of the requirements (see [Basic-website-characteristics](https://hackmd.io/s8LmceVRRf-23iTKCy7X8A#Basic-website-characteristics)). Currently we show the protocol text not as-is (the markdown) but with some additional metadata at the right. But printing will be done for the main text only (left). Actually we may print only required content automatically using default print feature.

> More useful action buttons could be in place of the print button, for example a link to (in the case of the json example) the Node website or a 'copy code to clipboard' button so somebody could paste straight into their terminal.

Node JSON here is just an example (probably confusing). As stated in the first Item, the text there is just a protocol definition markdown. If you see any reasonable actions we can add in this case - we are happy to consider.

**Comment by dhh1128** [comment]
I would like to see these mockups on mobile. I tried the link on my mobile phone but got a view identical to the one I saw on my laptop.

## Main Page

Even in a wide viewscreen like a desktop/laptop provides, I think 4 columns is too much. I'd merge the text from columns 1 and 2 so we only have 3 things to read.

The hyperlinks in the title text are a distraction. I'd remove them. ("Find" is already offered by the search box immediately below this title, so is unnecessary. Publish might be more interesting. But let's find a way to offer that without hyperlinking from the header.)

Let's drop the tag line from the icon, top left.

The tag line under the title, "communicate securely and privately" needs to differentiate DIDComm from Signal/Noise, WhatsApp, and so forth. I suggest "communicate on any transport, with your own security and privacy on your own terms". See https://daniel-hardman.medium.com/security-silos-and-sovereignty-522e30bb8eb4

The verbiage "publish a protocol" (title) and "register a protocol" (column text below) is inconsistent. I suggest we go with "publish".

I suggest we change column headers from questions to action phrases that are clickable. So the first column title could go from "What is DIDComm?" to "Learn about DIDComm"; the second column would become "Learn about Protocols", and the third would be "Publish a Protocol." But I also wonder if we should reorder these so "Register" is first. Not sure about that last suggestion.

I would drop the "Recently Added" stuff. Keep the main page simpler. (You could make the default browse view be by recency, if you want. Or not. Up to you.)

## Search Results

I quite like this page. Minor suggestions: the "Date" filters in the side bar are going to be the least useful -- so put them at the bottom. Generally, put the most useful/likely choices at the top. So in Status, "Production" should be at the top and "Proposed" at the bottom; in License, permissive licenses at the top and proprietary at the bottom.

>the inline emojis sit poorly... more readable as larger icons that stack above the header texts

Agreed. 

>The colour contrast between the yellow and gradient background is low (3.5) which is not great for accessibility and poor readability for all. I would suggest swapping out the gradient

Agreed. I did the gradient on the current DIDComm.org website. Don't feel bound to keep that. The only visual element of the current website that is meaningful is the logo. It has some symbolic meaning (showing people, orgs, and IoT devices linked together), but is not truly sacred, either. The color, in particular could be changed. (I have the logo as a vector, btw, if you only have .png of it.)

>multi-language support

Agree with Alex. Nice to have, but not a requirement at the moment. Are pypi.org, npmjs.org, and Maven Central localised? If we could plan for localisation, that'd be great -- but I would only spend time on laying the foundation for l10n if the time is very minimal. 

As far as American English: this text was written by me when I was working for an American company and had not challenged my assumptions much. The choice ought to be made by the people at DIF who will maintain it when we're done.

>As for the current text itself (in English), it's mostly take from existing didcomm.org website. I think we may change/adjust it

Yes. All this text was written by me, and none of it is particularly wonderful. The major change I'd make is to follow the "less is more" principle, and try to make the site less wordy overall. Changing the text is not really a developer task; the people at DIF who maintain this website should be able to submit a PR that easily updates the main text when we are done.

**Comment by dhh1128** [comment]
>Being able to print the protocol definition is part of the requirements

Yes. But that's mainly a CSS requirement, in my mind -- not so much a requirement that there be a "print" button anywhere.

**Comment by ashcherbakov** [comment]
Thank you for the comments. 
We've been working on the Mock-ups for Mobile, and will share them as soon as they are available. We just decided to share the web version for some early feedback.

**Comment by sergprokopenko** [comment]
Hello, thank you for feedback!

> * On the homepage the inline emojis sit poorly against the line height of the text and I would argue have minimal useful impact there at that scale and reduce readability. They may be more readable as larger icons that stack above the header texts, eg:

The icons height is 32px. The line height of $font-title-2 is 32px, and 40px for $font-large-title. We can increase icons for the large title. Of course icons above the header texts is more classical composition, but I took trend from here
https://uxdesign.cc/a-guide-to-the-modern-minimal-ui-style-531ac1e9fbfe


> * The colour contrast between the yellow and gradient background is low (3.5) which is not great for accessibility and poor readability for all. I would suggest swapping out the gradient for something more harmonious with the rest of the site. It feels like a little bit of an arbitrary choice at the minute

I used triangular color scheme, where yellow is from logo. We can make them darker to increase the contrast
![image](https://user-images.githubusercontent.com/42004425/126772513-dbfa72ee-515b-4186-a682-67633672fd05.png)



> * I find the positioning of the search bar in the centre of the header odd as it jumps from being a page element on the Homepage. This may also be difficult to translate into a consistent mobile experience. I would left align it over the first column of pages.

The idea was to make the header adaptive like amazon has
![image](https://user-images.githubusercontent.com/42004425/126773506-32e2953f-3ae6-44d5-808c-0b340fe22bbb.png)



> * The protocol page needs a back button to return to the search results. Persisting search results would be desirable.

It would be good, thanks

**Comment by sergprokopenko** [comment]
Hello, thank you for feedback!

> The hyperlinks in the title text are a distraction. I'd remove them.

Do you think it looks like links? I made links blue. Yellow text in the title is just an emphasis

**Comment by ashcherbakov** [comment]
@dhh1128 
> I would drop the "Recently Added" stuff. Keep the main page simpler.

Actually this element was proposed by me. I believe all similar sites (pypi, npm, creates etc.) have similar elements. There are also usually "most popular" elements, but currently we are not going to support popularity measurements, so we skip this.

We are open to removing the "Recently Added" element, but I like it being there. It shows that the site is "live", active and used by people. Also it can give a brief understanding what protocol is about and what examples can be right from main page.

One of the option is that we may not add these feature at the beginning (when probably we will not have a lot of published protocols), but add it (probably with "popular protocols") when we get more adaption.

What do you think @dhh1128 @ajslack @sergprokopenko @Merciful12 @vimmerru ?

**Comment by dhh1128** [comment]
Yes, that element will make more sense when we have more content. In the first few months of the website, there may only be a half dozen protocols, and they won't change -- so I was thinking it might be better to suppress.

If you want to keep it, perhaps we could rename it to be "Featured Protocols", where we decide what's featured based on the recency. But that way, if the most recent protocol is 3 months old, it won't look bad for the site.

**Comment by ashcherbakov** [comment]
Yes, that makes sense. What about the following: let's keep it in the UI mockups for future references, but don't implement for the first release of the site. We can create a ticket in the backlog for this.

**Comment by ashcherbakov** [comment]
@dhh1128 
> I suggest we change column headers from questions to action phrases that are clickable. So the first column title could go from "What is DIDComm?" to "Learn about DIDComm"; the second column would become "Learn about Protocols", and the third would be "Publish a Protocol." But I also wonder if we should reorder these so "Register" is first. Not sure about that last suggestion.

Where should the action phrases link to? 
- The DID Comm spec page for Learn about DIDComm?
- A page with instruction (didcomm.org github?) for Publish (Reguster) a Protocol?
- What should be a link for "Learn about Protocols"?

What about the text in the columns? Do you suggest to
- keep the text in the columns as-is, 
- OR keep the text in the columns as-is but remove the links from there (as it will be part of the header itself)
- OR remove the text at all keeping only clickable action phrases?

**Comment by dhh1128** [comment]
>Where should the action phrases link to? The DID Comm spec page for Learn about DIDComm? A page with instruction (didcomm.org github?) for Publish (Reguster) a Protocol?

Yes and yes.

>What should be a link for "Learn about Protocols"?

For now, https://github.com/hyperledger/aries-rfcs/blob/master/concepts/0003-protocols/README.md

>What about the text in the columns?

Keep the text as-is, but finish it with "...more", where "more" goes to the same place as the clickable header?

**Comment by sergprokopenko** [comment]
> Keep the text as-is, but finish it with "...more", where "more" goes to the same place as the clickable header?

Links "more" is bad practice. Better when link expresses the essence. Is there a way to intergate them into the text like it was?
![image](https://user-images.githubusercontent.com/42004425/126809751-a89142fa-0834-48fc-b396-aa3f1c2afa97.png)

Also I corrected composition here if trend from my link above is not appropriate

**Comment by dhh1128** [comment]
>Is there a way to integrate them into the text like it was?

Yes; make the phrases like "Learn about DIDComm" into hyperlinks. Then we can just drop the "...more" part at the end.

**Comment by dhh1128** [comment]
>Is there a way to integrate them into the text like it was?

Yes; make the phrases like "Learn about DIDComm" into hyperlinks. Then we can just drop the "...more" part at the end.

**Comment by ashcherbakov** [comment]
@dhh1128 @ajslack 
Please find updated versions of UI mock-ups:
1) Web: [link](https://www.figma.com/proto/YlEZDRezhuEoUUiAzFn22R/didcomm.org?page-id=28%3A0&node-id=1645%3A4010&viewport=3157%2C-487%2C0.5&scaling=min-zoom&starting-point-node-id=1645%3A4010&show-proto-sidebar=1)
2) Mobile: [link](https://www.figma.com/proto/YlEZDRezhuEoUUiAzFn22R/didcomm.org?page-id=28%3A0&node-id=1586%3A4027&viewport=1681%2C-477%2C0.25&scaling=min-zoom&starting-point-node-id=1586%3A4027)

**Comment by ajslack** [comment]
Thank you for the updates. A few comments:

**Homepage**

- I prefer the revised composition @sergprokopenko posted in the chat above for as the stacked icons, these will help break up the otherwise text heavy page and allow for better readability. This is especially evident when scrolling the mobile mock-up.

- On the mobile site the title has an additional margin (perhaps carried over from the desktop site?) that makes the line width very slim and incongruous. Would suggest removing additional margin here.
![Uploading 127282804...]

- Highlighting of Learn about **DIDComm**, Learn about **Protocols**, **Publish** a Protocol is inconsistent. Suggest highlighting the action term - the first becomes **Learn** about DIDComm, the second one could change to **Explore** Protocols? @dhh1128 ?

**Search results**

- Clicking on filters in mobile opens a full screen side pane. This is really awkward - would suggest an in-line accordion for better UX, especially on mobile.

**Protocol Info Page**

- Excessive spacing and use of stacking on top elements. Back button, title, action button and hashtags could be much better condensed or even have some bought inline to improve readability.

- Stand by previous comment that the 'print' action button is an odd addition. Would suggest a 'copy code to clipboard' or similar such useful action

**Comment by sergprokopenko** [comment]
> * On the mobile site the title has an additional margin (perhaps carried over from the desktop site?) that makes the line width very slim and incongruous. Would suggest removing additional margin here.
>   ![Uploading 127282804...]

Sorry, I don't see your uploading attachment and didn't get what margin are you talking about

**Comment by dhh1128** [comment]
Thanks for the updates. I feel like the new mockups are a noticeable step forward. It's exciting.

General question: How easy will it be to change content and visual appearance once the site is built? *If I can change it by simply committing a slight change to static files (e.g., tweaking a CSS, adding or removing a few words from a &lt;p&gt; in the HTML), then a lot of the feedback I'm giving below can be delayed, and we can experiment easily until we're happy. I'd be content to close the issue after this round of feedback. But if it takes special knowledge or difficult processes to update the site, then we have to get it right as much as possible, right now...*

<hr>

I think the tweak to colors/gradient is a nice improvement to the contrast. I don't have expertise in color contrast and the like, so I will let @ajslack comment about whether he's happy with the new gradient, or whether he continues to feel like it would be better to just eliminate the gradient entirely. For myself, I think I could be happy with the gradient in its new form, or with a flat blue like the one on the left side of the gradient.

I really dislike the embedded hyperlinks in the page title. I said this before, and I feel like this feedback was lost or not noticed, so I'm going to reiterate. There is no point to making "Find" into a hyperlink, when it just links to the very search box that appears immediately below it. Both are "above the fold" -- so clicking "Find" isn't even going to make the search box scroll into view. Plus, "Find" is inconsistent with the verb "search" right below it. And the inline graphics in the title are distracting and jarring, not helpful.

I also think there's a problem with being too wordy. (This is a critique of my own work, since I am the one who wrote most of those words. :-) A person has to read too much to get any value from the page. This is particularly true in the mobile mockup, where the search feature is the only thing above the fold. Ideally, I think the page would have content only above the fold, requiring no scrolling at all. I don't know if it's practical -- but do you agree that it might be nice? That would mean that I need to propose text that is shorter and sweeter. I'll do that below. But we might also want to make the top colored rectangle with the downward curve into something that could take up less space if we have pressure on vertical real estate; right now that rectangle is taking up a lot of space without providing a lot of value...

### Specific proposals about simplifying the main page

* Change title to just "Share protocols built on DIDComm". ("Share" subsumes both find/search and publish -- and if we drop the hyperlinks, combining the words is fine. "On" is shorter and sounds more casual than "atop". "Decentralized" is jargon/buzzward and creates dissonance for either Brits or Americans depending on how it's spelled. Less is more.)

* How about changing the tagline underneath title to "powerful, high-trust, self-sovereign interactions over any transport"? It's a little shorter, and it is more accurate and meaningful. It's also not a sentence, so it doesn't need to be capitalized or have a period at the end.

* In the 3 columns at the bottom, I'd like the paragraphs to be approximately the same height. How about changing the first one, which is currently 8 lines including one blank, beginning with "DIDComm lets people", into a single paragraph of about 5 lines: "[Read](https://identity.foundation/didcomm-messaging/spec/) how to control your security and privacy, whether it's over the web, email, push notifications, QR codes, BLE, NFC, message queues, sneaker net... [Find libraries](https://github.com/decentralized-identity/didcomm-messaging#implementations) or [connect with DIDComm's authors](https://github.com/decentralized-identity/didcomm-messaging/issues)."

* Earlier I made the suggestion to make the headers of the 3 columns clickable. Now that this implemented in the mockup, I don't like it. I was imagining the whole header clickable rather than just a single word of each header -- but even if it were changed to the way I imagined, I think it's going to be suboptimal. I was wrong. Instead, let's keep the hyperlinks in the running text beneath each header, and remove the link in the header. In the middle paragraph, we can make the phrase "built atop DIDComm" link to the place where the header used to link.

* The icons don't make sense. A stack of books is mildly suggestive of "publish", but really, I think we want to use the same icon for that verb that mobile apps use for sharing: something like [one of these](https://stock.adobe.com/at/search?k=share+button&asset_id=343509272), I would think. (And since we're changing the language in the page title to say "Share" instead of "Find and publish", let's just change the header text in the third column to say "Share a protocol", too; that makes the icon work even better. An icon for "learn" might be like [these](https://duckduckgo.com/?q=learn+icon&atb=v271-1&iax=images&ia=images), instead of being a magic wand. And the icon for a protocol should probably draw inspiration from [these](https://docs.google.com/presentation/d/15UAkh_2WfDk7wlto7pSL7YU9NJr_XVMgGAOeNIRbzK8/edit#slide=id.p).

* The "Browse all protocols" link feels like an anomaly, visually. It's in a weird place, and it looks different than everything else. What if we fixed this by moving  by making it a button to the right of "Search", just labeled "Browse"? This would also save vertical screen space.

**Comment by dhh1128** [comment]
Let's also change the header text for the middle column to "Build a Protocol" so we're not using "Learn" twice.

**Comment by dhh1128** [comment]
More feedback. On normal search results page:

![main search results](https://i.ibb.co/SNcFVMk/feedback-on-search-results.png)

On mobile view of search results or individual protocol view:

![mobile](https://i.ibb.co/RYt9c5G/feedback-on-mobile-search-results.png)

**Comment by ajslack** [comment]
> > * On the mobile site the title has an additional margin (perhaps carried over from the desktop site?) that makes the line width very slim and incongruous. Would suggest removing additional margin here.
> 
> >   ![Uploading 127282804...]
> 
> 
> 
> Sorry, I don't see your uploading attachment and didn't get what margin are you talking about

Apologies, hopefully this should be embedded now -

![image](https://user-images.githubusercontent.com/75382216/127330728-b3fee568-3c84-40dd-bb13-41f5f3067b7c.jpeg)

**Comment by ajslack** [comment]
Agree with @dhh1128 we should move to close this issue. I think generally it is heading in a good direction. I include a few responses below to other comments.

>I think the tweak to colors/gradient is a nice improvement to the contrast. I don't have expertise in color contrast and the like, so I will let @ajslack comment about whether he's happy with the new gradient, or whether he continues to feel like it would be better to just eliminate the gradient entirely. For myself, I think I could be happy with the gradient in its new form, or with a flat blue like the one on the left side of the gradient.
> 
Presume this is something that can easily be updated so not urgent or blocking. Though I think a flat colour would work better in the context of the rest of the design, a flat blue would be ok. The contrast is now better.
> 
> I really dislike the embedded hyperlinks in the page title. I said this before, and I feel like this feedback was lost or not noticed, so I'm going to reiterate. There is no point to making "Find" into a hyperlink, when it just links to the very search box that appears immediately below it. Both are "above the fold" -- so clicking "Find" isn't even going to make the search box scroll into view. Plus, "Find" is inconsistent with the verb "search" right below it. And the inline graphics in the title are distracting and jarring, not helpful.
> 
Agree I don't think the inline graphics work and would suggest removing.

> * Change title to just "Share protocols built on DIDComm". ("Share" subsumes both find/search and publish -- and if we drop the hyperlinks, combining the words is fine. "On" is shorter and sounds more casual than "atop". "Decentralized" is jargon/buzzword and creates dissonance for either Brits or Americans depending on how it's spelled. Less is more.)
> 
> * How about changing the tagline underneath title to "powerful, high-trust, self-sovereign interactions over any transport"? It's a little shorter, and it is more accurate and meaningful. It's also not a sentence, so it doesn't need to be capitalized or have a period at the end.
> 
Agree with both, shorter, clearer and to the point.
> 
> * The icons don't make sense. A stack of books is mildly suggestive of "publish", but really, I think we want to use the same icon for that verb that mobile apps use for sharing: something like [one of these](https://stock.adobe.com/at/search?k=share+button&asset_id=343509272), I would think. (And since we're changing the language in the page title to say "Share" instead of "Find and publish", let's just change the header text in the third column to say "Share a protocol", too; that makes the icon work even better. An icon for "learn" might be like [these](https://duckduckgo.com/?q=learn+icon&atb=v271-1&iax=images&ia=images), instead of being a magic wand. And the icon for a protocol should probably draw inspiration from [these](https://docs.google.com/presentation/d/15UAkh_2WfDk7wlto7pSL7YU9NJr_XVMgGAOeNIRbzK8/edit#slide=id.p).
> 
Agree the icons don't work as is but I think again this is something that should be easy to swap out as we go.
>

**Comment by sergprokopenko** [comment]
Hello,

We updated mobile and web prototypes

https://www.figma.com/proto/YlEZDRezhuEoUUiAzFn22R/didcomm.org?page-id=28%3A0&node-id=1763%3A5556&viewport=7058%2C-1564%2C1&scaling=min-zoom&starting-point-node-id=1763%3A5556&show-proto-sidebar=1

https://www.figma.com/proto/YlEZDRezhuEoUUiAzFn22R/didcomm.org?page-id=28%3A0&node-id=1645%3A4010&viewport=7058%2C-1564%2C1&scaling=min-zoom&starting-point-node-id=1645%3A4010&show-proto-sidebar=1

Also here some our thoughts:

1. We suggest to keep the logo and to remove button back. We need logo for transition to the home page. So we have common header (logo and search). Button back duplicates the functionality of the browser
![image](https://user-images.githubusercontent.com/42004425/127529782-2bca7073-371e-4cd2-9124-c554f77fd65d.png)

2. We suggest to keep full screen for the filters. It is good practice for mobile.
"Create a modal experience only when it’s critical to focus people’s attention on making a choice or performing a task that’s different from their current task"
https://developer.apple.com/design/human-interface-guidelines/ios/app-architecture/modality/

3. If we made browse and search in line, they would be connected more strongly. 2 lines show the separation and better for F-shaped reading pattern

**Comment by sergprokopenko** [comment]
Also we compressed all mobile mockups

And this is an option for header and icons color
![image](https://user-images.githubusercontent.com/42004425/127530521-fb9c688c-6fae-4e2b-bf95-b2990ffee464.png)

**Comment by sergprokopenko** [comment]
![image](https://user-images.githubusercontent.com/42004425/127623360-f2d46353-4b80-4e1d-b95b-318db9ce08e9.png)
![image](https://user-images.githubusercontent.com/42004425/127623434-70c57ffc-6a62-4f95-a0d2-8bc2e32c84c4.png)

Right, this is a center of 12 columns grid

**Comment by ashcherbakov** [comment]
@dhh1128 @ajslack 
Can you please let us know if the new mockups seem better for you, and what major issues/changes you see.
Also, what colour schema (gradient + blue icons or no-gradient + yellow icons) do you like more? (Personally, I like the option with yellow icons). 

In general, we are going to start UI implementation based on the proposed UI design. If there any minor issues, findings or recommendations found, I believe we can fix/apply them right in the code, as that's something that can be changed quite easily.

**Comment by sergprokopenko** [comment]
> The problem with this reading is that browse and search ARE strongly connected on this particular website. On some websites, maybe not -- but on this website, browse is just search with no filters/criteria. They take you to the same place. There is no point in separating them. I don't believe this separation makes sense. And the layout that's "better for F-shaped reading patter" is also quite ugly.

Ok, we got it. Do you think some of the mockups below are better?

1 
![image](https://user-images.githubusercontent.com/42004425/127886224-27979dce-9797-4260-9789-3f847397e76e.png)

2 
![image](https://user-images.githubusercontent.com/42004425/127886254-0c782cca-ff0b-4bdb-a81a-2dd1883012d8.png)

3 
![image](https://user-images.githubusercontent.com/42004425/127886299-27566d2e-0ccd-4028-85a3-dc1f9c19db67.png)

**Comment by dhh1128** [comment]
All of these are improvements. I like #1 and #2 better (the blue region is good at the top). I like #1 slightly better than #2 -- but #2 is also pretty good.

**Comment by dhh1128** [comment]
I'm just noting that, although there may be minor issues outstanding, I don't consider anything of them significant enough to keep this ticket open. Thanks for all the good design work -- and full speed ahead on implementation.
